jalib/jconvert.h

```diff
@@ -40,7 +40,7 @@ StdLibEC(const dmtcp::string &s, bool strict)
 {
```

Comment:
I don't see where `StdLibEC<a, b, c>()` is used.  I only see `StdLibEC<a, b>()`.  If `StdLibEC<a, b, c>()` is not used, let's just delete it.  (Or can you point me to where it's used?)

Reply:
The <a,b> functions are used for `strtof/strtold` while the <a,b,c> variant is used for `strol` variants.